app/javascript/components/map/map-reducers.js

```diff
@@ -1,8 +1,8 @@
 export const initialState = {
   loading: false,
   error: false,
+  options: {},
```

Comment:
I think we are starting to get messy with the map state. Options and settings to me are the same thing. Let chat to decide on how we should structure this.